Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM node:12.16.1-alpine as builder
+WORKDIR /opt/auth-service
+
+RUN apk add --no-cache -t build-dependencies git make gcc g++ python libtool autoconf automake \
+    && cd $(npm root -g)/npm \
+    && npm config set unsafe-perm true \
+    && npm install -g node-gyp
+
+COPY package.json package-lock.json* /opt/auth-service/
+
+RUN npm ci
+COPY src /opt/auth-service/src
+COPY test /opt/auth-service/
```

Comment:
Do we need this here?

Reply:
we need `src` to be able to run the typescript source code with ts-node
we need `test` to be able to run the tests inside the container

Reply:
Sure. We actually don't normally run the unit tests inside of our containers, but outside before we build them. That way we don't need the `devDependencies` as well.

I'm happy to leave this for now, and we can discuss later on.